framework/validators/FileValidator.php

```diff
@@ -156,6 +177,9 @@ public function init()
         if ($this->tooMany === null) {
             $this->tooMany = Yii::t('yii', 'You can upload at most {limit, number} {limit, plural, one{file} other{files}}.');
         }
+        if ($this->tooFew === null) {
+            $this->tooFew = Yii::t('yii', 'You can upload at less than {limit, number} {limit, plural, one{file} other{files}}.');
```

Comment:
`You can upload at less than {limit, number} {limit, plural, one{file} other{files}}.` -> `You should upload at least {limit, number} {limit, plural, one{file} other{files}}.`

Reply:
I fixed it